docs/advanced/advanced.md

```diff
@@ -192,6 +192,49 @@ The contents of `extra_navbar` will be inserted into your page's HTML after
 all other html content.
 
 
+## Working on Windows
+
+As of May 27, 2020, there are three open issues that require Windows-specific workarounds:
```

Comment:
I think we should add something to the effect of "note that behavior on Windows may still be unstable, and these workarounds are community tips but are not guaranteed to solve your problem"